Extension/src/commands.ts

```diff
@@ -11,6 +11,7 @@ class TemporaryCommandRegistrar {
     // Used to save/re-execute commands used before the extension has activated (e.g. delayed by dependency downloading).
     private delayedCommandsToExecute: Set<string>;
     private tempCommands: vscode.Disposable[]; // Need to save this to unregister/dispose the temporary commands.
+    private isDisabled: boolean;
```

Comment:
Since this is just about disabling the language server, I think this variable name should reflect that.

Reply:
Sure.